app/javascript/packs/globals.js

```diff
@@ -57,3 +57,5 @@ require('codemirror/mode/xml/xml.js');
 require('codemirror/mode/yaml/yaml.js');
 require('codemirror/lib/codemirror.css');
 require('codemirror/theme/eclipse.css');
+import * as move from '../helpers/move.js';
+window.move = move;
```

Comment:
@ZitaNemeckova could we include this inside of the `ManageIQ` global variable?

Reply:
Resolved :)